includes/shortcodes.php

```diff
@@ -107,11 +107,49 @@ function tsml_next_meetings($arguments)
 });
 
 //output a react meeting finder widget https://github.com/code4recovery/tsml-ui
-function tsml_ui()
+function tsml_ui($arguments)
```

Comment:
Small thing, would add default `tsml_ui($arguments = array())` - just in case someone's using this method somewhere else, and without the param it'll break.

Reply:
good call 👍